app/helpers/container_service_helper/textual_summary.rb

```diff
@@ -28,7 +28,7 @@ def textual_group_port_configs
   end
 
   def textual_group_relationships
-    items = %w(ems container_project container_groups)
+    items = %w(ems container_project container_groups container_routes)
```

Comment:
Order here is wrong, it should be:

    items = %w(ems container_project container_routes container_groups)